rest/rest-cli/src/test/java/org/ow2/proactive_grid_cloud_portal/cli/CommandSetTest.java

```diff
@@ -79,7 +79,11 @@ public void testThatArgNamesMatchNumberOfArgs() throws ParseException, IllegalAc
 
             if (!argNames.trim().isEmpty()) {
                 nbArgsBasedOnName = argNames.split(" ").length;
-                nbArgsBasedOnName = nbArgsBasedOnName - optionals;
+                if (entry.hasArgs()) {
```

Comment:
(Question) Do we consider in this if-condition cases with non optional+ optional args?